src/app/Library/Auth/AuthenticatesUsers.php

```diff
@@ -199,4 +204,26 @@ protected function guard()
     {
         return Auth::guard();
     }
+
+    private function verifyUserBeforeLogin(Request $request): Response|\Illuminate\Http\RedirectResponse|\Illuminate\Routing\Redirector
+    {
+        $user = $this->guard()->user();
+
+        if ($user->email_verified_at) {
+            // if the user is verified send the normal login response
+            return $this->sendLoginResponse($request);
+        }
```

Comment:
We don't actually need to return the loginResponse here, right? It will be called next anyway.
```suggestion
        // if the user is already verified, do nothing
        if ($user->email_verified_at) {
            return;
        }
```

Reply:
We actually don't.